nautobot_netbox_importer/diffsync/adapters/netbox.py

```diff
@@ -79,6 +91,10 @@ def load(self):
             diffsync_model = getattr(self, modelname)
             self.logger.info(f"Loading all {modelname} records...")
             content_type_label = diffsync_model.nautobot_model()._meta.label_lower
+            # Handle a NetBox vs Nautobot discrepancy - the Nautobot target model is 'users.user',
+            # but the NetBox data export will have user records under the label 'auth.user'.
+            if content_type_label == "users.user":
```

Comment:
Just food for thought: 

I noticed you're having to do this in two places: here and in `nautobot_netbox_importer/diffsync/models/contenttypes.py`.  Perhaps this is a rare edge case, but seems like maybe a central mapping of content-types might be useful if it comes up again.

Reply:
I haven't been able to come up with a good way to de-duplicate it that I'm happy with, because the two are subtly different - one is the `app_label` and `model` attributes on an individual `ContentType` record, the other (here) is the combined model label on the `User` model class. I could create a shared mapping dict somewhere but the logic to make use of it would still be quite different in these two cases. Agreed though that this may be a recurring issue to solve in the future.